src/app/containers/Article/index.jsx

```diff
@@ -11,46 +11,70 @@ import ArticleMain from '../ArticleMain';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import ConsentBanner from '../ConsentBanner';
+import { GhostWrapper, GridItemConstrainedMedium } from '../../lib/styledGrid';
 
 const logger = nodeLogger(__filename);
 
-/*
-  [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
-*/
+const Container = ({ children, service }) => (
+  <Fragment>
+    <GlobalStyle />
+    <ServiceContextProvider service={service}>
+      <Helmet>
+        <link rel="manifest" href={`/${service}/articles/manifest.json`} />
+      </Helmet>
+      <ConsentBanner />
+      <HeaderContainer />
+      {children}
+    </ServiceContextProvider>
+  </Fragment>
+);
+
+Container.propTypes = {
+  children: node.isRequired,
+  service: string.isRequired,
+};
+
 const ArticleContainer = ({ loading, error, data, bbcOrigin }) => {
-  if (loading) return 'Loading...'; /* [1] */
+  if (loading) {
+    return (
+      <Container service="news">
+        <main role="main">
+          <GhostWrapper>
+            <GridItemConstrainedMedium />
+          </GhostWrapper>
+        </main>
+      </Container>
+    );
+  }
+
   if (error) {
     logger.error(error);
-    return 'Something went wrong :(';
+    return (
+      <Container service="news">
```

Comment:
Should this be hardcoded to `news`? I haven't checked but even when loading I would of thought that `data.service` still had the right value for `service` in the react router regex match

Reply:
This error page should work when our application fails to load any data. I agree that I don't want to pin to use 'news'.